src/PermissionServiceProvider.php

```diff
@@ -123,6 +123,10 @@ protected function registerBladeExtensions($bladeCompiler): void
         $bladeCompiler->directive('elserole', fn ($args) => "<?php elseif({$bladeMethodWrapper}('hasRole', {$args})): ?>");
         $bladeCompiler->directive('endrole', fn () => '<?php endif; ?>');
 
+        $bladeCompiler->directive('haspermission', fn ($args) => "<?php if({$bladeMethodWrapper}('hasPermissionTo', {$args})): ?>");
```

Comment:
`hasPermissionTo` throws an exception, must be `checkPermissionTo`
https://github.com/spatie/laravel-permission/blob/a10d72d40ec41e3e65fe390864667f4c2dec2fb7/src/Traits/HasPermissions.php#L254-L261